src/autocomplete/databases/yql/yql-autocomplete.ts

```diff
@@ -46,6 +47,33 @@ const tokenDictionary: TokenDictionary = {
     SELECT: YQLParser.SELECT,
 };
 
+function getUniqueTableSuggestions(suggestions: Table[] = []): Table[] {
+    const suggestionsMap = suggestions.reduce(
+        (acc, table) => {
+            const aliases = acc[table.name] ?? new Set();
+            if (table.alias) {
+                aliases.add(table.alias);
+            }
+
+            acc[table.name] = aliases;
+            return acc;
+        },
+        {} as Record<string, Set<string>>,
+    );
+    return Object.keys(suggestionsMap).reduce((acc, el) => {
```

Comment:
we don't use shortened variable names in our code (regarding ```el```)
also, it's better to give a specific name rather than something generic
e.g. ```tableName```